### ISSUE TITLE
[Feature request] Ability to set/control the sequence track strand in Batch Script/Socket

### ISSUE THREAD

**Issue by bbreton** [igvteam/igv#836]
https://software.broadinstitute.org/software/igv/sequence_track_options

In the sequence track I would like to be able to set the strand (+/-) which is currently controlled in the GUI only with "Flip strand" when using the Batch Script/Socket control methods.

My specific use case for this issue is I want it to set it to the correct strand when using a Batch Script/Socket control doing snapshots so it has the right direction. Explicitly setting the strand makes more sense for this use case, but I could work with the ability to flip it as long as I could tell the starting state in case that is easier to do or fits other uses better.

Also (perhaps this should be a separate issue) the current strand of the sequence track isn't saved in the Save Session xml file as I looked into using loading a session as a workaround for no Batch Script command to control it. I feel like it would make sense for it to be saved in the session.

My testing was done with 2.8.9.

Thank you IGV team for all your work and contributions to the community!

**Comment by jrobinso** [comment]
Yes a command would make sense.   The second issue is a bug, that should be in the session.   Will look at both.

**Comment by jrobinso** [comment]
@bbreton is there anything still outstanding (i.e. left to do) here?
